docs/shared/angular-tutorial/angular-monorepo.md

```diff
@@ -981,7 +981,7 @@ Learn more about how to [enforce module boundaries](/core-features/enforce-modul
 
 ## Setting Up CI
 
-Without adequate tooling, CI times tend to grow exponentially with the size of the codebase. Nx helps decrease the average CI time with the [`affected` command](/nx-cloud/features/affected) and Nx Cloud's [distributed caching](/nx-cloud/features/remote-cache). Nx also [decreases the worst case CI time](/nx-cloud/concepts/dte) with Nx Cloud's distributed task execution.
+Without adequate tooling, CI times tend to grow exponentially with the size of the codebase. Nx helps reduce waste in CI with the [`affected` command](/nx-cloud/features/affected) and Nx Cloud's [remote caching](/nx-cloud/features/remote-cache). Nx also [efficiently parallelizes tasks across machines](/nx-cloud/concepts/parallelization-distribution) with Nx Cloud's distributed task execution.
```

Comment:
are we sure "reduce waste" is the language ppl associate with CI? Usually you just talk about timing

Reply:
Good call out.  I changed it to "reduce wasted time in CI".  Which still accurately describes whats happening and phrases it in a way that people understand that its faster.